pom.xml

```diff
@@ -402,7 +402,7 @@
         <dependency>
             <groupId>net.snowflake</groupId>
             <artifactId>snowflake-jdbc</artifactId>
-            <version>3.13.14</version>
+            <version>3.13.23</version>
```

Comment:
Have you try removing this dependency because we already have it in the ingest SDK?

Reply:
Yes. I have and it works, but I felt that it would be cleaner to have it as part of another commit that's intentional on removing it and to document why. It also becomes kind of a mystery for someone to figure out what JDBC version we're using in the Kafka connector by simply glancing over the pom.xml file. Not sure, what do you think?